exec-unittests

```diff
@@ -161,43 +166,57 @@ if [[ -z "${subtst}" || "${subtst}" == "python" ]]; then
     fi
 fi
 
-_subtst_list="tool-scripts/datalog tool-scripts/postprocess tool-scripts util-scripts bench-scripts"
+_agent_subtst_list="tool-scripts/datalog tool-scripts/postprocess tool-scripts util-scripts bench-scripts"
 
 _para_jobs_file="${_toxenvdir}/agent-legacy-jobs.lis"
 trap "rm -f ${_para_jobs_file}" EXIT INT TERM
 
 for _major in ${major_list}; do
-    # Verify the Agent or Server Makefile functions correctly.
-    if [[ "${_major}" == "agent" || "${_major}" == "server" ]]; then
-        verify_make_source_tree ${_major} || rc=1
-    fi
-
     if [[ "${_major}" == "agent" ]]; then
-        # The parallel program is really cool.  The usage of `parallel` is
-        # internal and automated; only test code depends on this tool, and we,
-        # as developers, have viewed the citation and are justified in
-        # suppressing future displays of it in our development processes (use of
-        # --will-cite below).
-
-        let count=0
-        for _subtst in ${_subtst_list}; do
+        if [[ "${subtst:-legacy}" == "python" ]]; then
+            continue
+        fi
+        for _subtst in ${_agent_subtst_list}; do
             if [[ "${subtst:-legacy}" == "legacy" || "${subtst}" == "$(basename ${_subtst})" ]]; then
                 echo "${_major} ${_subtst} ${posargs}"
-                (( count++ ))
             fi
         done > ${_para_jobs_file}
```

Comment:
I think it's a bad idea to have two different variables whose names differ only in the leading underscore -- that seems like a setup for a pratfall.

Oh, and, let's check my `bash` fu again...how do you like this?
```
        for st in ${_agent_subtst_list}; do
            st_name=$(basename ${st})
            if [[ "${subtst:-${st_name}}" == "${st_name}"]]; then
                echo "${_major} ${st} ${posargs}"
            fi
        done > ${_para_jobs_file}
```
😁